network_simulator/common.py

```diff
@@ -0,0 +1,15 @@
+"""This file contains definition of constants.
+"""
+
+ACK_PACKET_SIZE_BITS = 512
+"""Size of :class:`.AckPacket` is 512 bits."""
```

Comment:
Put comment above variable declaration. Same with the other variable declarations here.